client/scss/base/_typography.scss

```diff
@@ -52,45 +37,25 @@ p {
 
 ul,
 ol {
-  margin: 0.5rem 0 1.7rem 2rem;
-
   &:last-child {
     margin-bottom: 0;
   }
 }
 
-ul {
-  list-style-type: disc;
-}
-
-ol {
-  list-style-type: decimal;
-}
-
 li {
-  margin-bottom: 0.1em;
-
   &:last-child {
     margin-bottom: 0;
   }
 }
 
-em,
-i {
-  font-style: italic;
-
-  strong,
-  b {
-    font-weight: bold;
-  }
-}
-
-strong,
-b {
+/* Temporary until decide on font hosting solution */
+/*
+ * Wellcome font
+ */
+@font-face {
+  font-family: 'wellcome bold';
```

Comment:
This was just a quick and dirty way of getting them in, until we settle on a proper solution.

At the moment we own one font, 2 are available on fonts.com and one we have to get from somewhere else. It all needs rationalising.